chain/consensus/filcns/upgrades.go

```diff
@@ -2521,9 +2519,7 @@ func UpgradeActorsV14(ctx context.Context, sm *stmgr.StateManager, cache stmgr.M
 		return cid.Undef, xerrors.Errorf("migrating actors v14 state: %w", err)
 	}
 
-	if term.IsTerminal(int(os.Stderr.Fd())) {
-		fmt.Print(upgradeSplash)
```

Comment:
I think this one's fine .. we probably should have made it check `os.Stdout.Fd()` though; it just means that it's not going anywhere for anyone not running it in a console. which is great. If it goes into the log then it messes up people's logs with the big ugly waffle splash. Alternatively we could print the splash to stderr because it really is a throw-away cosmetic thing.